src/cf-tables/usage.md

```diff
@@ -692,3 +417,288 @@ highest to lowest)
   (last to first). These classes are added to the TH when sorting occurs.
 - Please note the importance of defining a THEAD and TBODY to preserve
   the table's header through sorting operations.
+
+
+## Responsive tables
+
+**Important notes:**
```

Comment:
This note is repeated in the paragraph below. It could be condensed to a single note in either spot.